.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*   @UpCloudLtd/devex
```

Comment:
@Darep, do you want to be included here?

Reply:
Sure!